plugin-offline/src/main/java/com/mapbox/mapboxsdk/plugins/offline/Constants.java

```diff
@@ -1,6 +1,6 @@
 package com.mapbox.mapboxsdk.plugins.offline;
 
-class Constants {
+public class Constants {
```

Comment:
if possible keep this package private

Reply:
we only have two public constants in here which can be moved into a separate constant file but I don't see the urgency. Places plugin exposes some constants also.